requirements.txt

```diff
@@ -55,3 +54,4 @@ bcrypt
 flask-bcrypt
 mock
 hdfs
+impyla
```

Comment:
Looks cool indeed! But the removal was breaking travis, but you found that out ;-)